assets/scss/licenses.scss

```diff
@@ -0,0 +1,47 @@
+.icon-license-pd {
+  background-image: url('../../assets/img/cc-icons/pd.svg');
```

Comment:
Do the paths still work if we use `~assets` instead of `../../assets`? If so, it will be cleaner plus we could drop the file into any directory without breaking the paths

Reply:
I have updated this. It used to be ../../ to get it working in Storybook, but since we are not using that actively, I have updated it in this file. I will create a technical debt task to update this in other CSS files.